terraform.tfvars.example

```diff
@@ -69,7 +69,7 @@ logAnalyticsWorkspaceName = ""
 logAnalyticsWorkspaceResourceGroupName = ""
 
 # The version of the AKS cluster.
-kubernetesVersion = "1.24.9"
+kubernetesVersion = "1.24.10"
```

Comment:
Did you maybe try to upgrade version to 1.26 and test these changes?

Reply:
I suggest to create a separate PR.

Reply:
done :see_no_evil: